src/assets/theme/customStyleBlocks/getSearchAutocompleteStyles.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Autocomplete styles specific to homepage searchbar (removes gray hover from dropdown option items)
+ */
+import { makeStyles } from '@material-ui/core/styles';
+import { COLOR_MAP } from 'common/colors';
+
+export const getSearchAutocompleteStyles = makeStyles(() => ({
+  option: {
+    '&[data-focus="true"]': {
+      backgroundColor: 'transparent',
+      color: `${COLOR_MAP.BLUE}`,
```

Comment:
nit: I think you don't need the string template here (`COLOR_MAP.BLUE` is already a string)